internal/workload/v1/resource.go

```diff
@@ -109,3 +109,35 @@ func determineSourceFileName(manifestFile string) SourceFile {
 
 	return sourceFile
 }
+
+func expandResources(path string, resources []*Resource) ([]*Resource, error) {
+	var expandedResources []*Resource
+
+	for _, r := range resources {
+
+		fileInfo, err := os.Stat(filepath.Join(path, r.FileName))
+		if err != nil {
+			return []*Resource{}, err
+		}
+
+		if fileInfo.IsDir() {
+			err := filepath.Walk(filepath.Join(path, r.FileName), func(p string, info os.FileInfo, err error) error {
+				if err != nil {
+					return err
+				}
+				if !info.IsDir() {
+					res := &Resource{FileName: filepath.Join(r.FileName, filepath.Base(p))}
```

Comment:
@lander2k2 also what would happen here if i was a consumer who accidentally left a `README.md` in my directory structure.  Should we filter out file types for resources and restrict to `.yaml` and `.yml` extensions?  Obviously this wouldn't be super prevelent if we point to individual files, but could be a consideration when we point at a directory.

Reply:
Agreed.  I will add a filter for this.